basket/settings.py

```diff
@@ -70,6 +61,15 @@ def path(*args):
     }
 DEFAULT_AUTO_FIELD = "django.db.models.AutoField"
 
+# CACHE_URL and RQ_URL are derived from REDIS_URL.
+REDIS_URL = config("REDIS_URL", None)
+if REDIS_URL:
+    REDIS_URL = REDIS_URL.rstrip("/0")
+    HIREDIS_URL = REDIS_URL.replace("redis://", "hiredis://")
+    # use redis for cache and rq.
+    os.environ["CACHE_URL"] = HIREDIS_URL + "/" + config("REDIS_CACHE_DB", "1")
```

Comment:
I'd have to look, but I think the reason we set these in the env rather than just keeping them settings is that celery would auto setup some things from the env in ways that wasn't easy via django settings. I don't think we have that same situation here do we? I could also be misremembering.